README.md

```diff
@@ -1,24 +1,25 @@
-![evidence-logo](/readme/evidence-logo.svg)
+<!-- TODO: replace -->
+![evidence-logo](sites/docs/static/img/evidence.svg)
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 
-# Business Intelligence for Modern Data Teams
+# Business Intelligence as Code
 
-Build a polished business intelligence system using only SQL and markdown. 
+Evidence is an open source, code-based alternative to drag-and-drop business intelligence tools.
 
-Evidence offers a modern development workflow for analysts and a polished user experience for decision-makers.
+You write SQL queries alongside markdown files and Evidence compiles them into a data website.
```

Comment:
"data website" isn't a term that I think people will have heard before. I would use "web app", or skip the web part and focus on the actual outputs: reports and analysis? 

e.g. 

"Evidence enables analysts [you] to build reports and analysis using SQL and markdown" or similar.